### PR TITLE
feat(css/compat): function notation

### PR DESCRIPTION
**Pull request by alexander-akait**
**Description:**

Port https://preset-env.cssdb.org/features/#color-functional-notation

**BREAKING CHANGE:**

No

**Related issue (if exists):**

No

### CODE REVIEW

File: crates/swc_css_compat/src/feature.rs [hunk 1]
```diff
@@ -6,5 +6,8 @@ bitflags! {
         const CUSTOM_MEDIA = 1 << 1;
         const MEDIA_QUERY_RANGES = 1 << 2;
         const COLOR_HEX_ALPHA = 1 << 3;
+        const COLOR_ALPHA_PARAMETER = 1 << 4;
+        const COLOR_SPACE_SEPARATED_PARAMETERS = 1 << 5;
+        const COLOR_LEGACY_RGB_AND_HSL = 1 << 5;
```
Comment (kdy1) [inline crates/swc_css_compat/src/feature.rs:11]: Typo? It's `1 << 5`

Comment (alexander-akait) [reply crates/swc_css_compat/src/feature.rs:11]: Yeah, I will fix

**Comment by alexander-akait** [comment]
@kdy1 Fixed

**Review by kdy1 — APPROVED** [review]
swc-bump:
 - swc_css_compat

**Pull request merged**